src/components/settings/SettingsSSOTab/tests/NewSSOConfigForm.test.jsx

```diff
@@ -459,14 +459,15 @@ describe('SAML Config Tab', () => {
       attr_username: 'foobar',
       other_settings: '',
       enabled: 'true',
-      debug_mode: 'true',
+      debug_mode: 'false',
       skip_hinted_login_dialog: 'true',
       skip_registration_form: 'true',
       skip_email_verification: 'true',
       send_to_registration_first: 'true',
-      automatic_refresh_enabled: 'true',
+      automatic_refresh_enabled: 'false',
     };
     mockUpdateProviderConfig.mock.calls[0][0].forEach((value, key) => {
+      console.log('WOW!', key);
```

Comment:
Should this be included in the merge or is it a dev artifact?

Reply:
hahaha oops!